src/CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ install(EXPORT ${LIB_EXPORT_NAME}
 
 # Intel MKL-DNN compat cmake files
 install(CODE "execute_process(COMMAND ${CMAKE_COMMAND}
-    -DDIR=${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/cmake
+    -DDIR=\$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/cmake
```

Comment:
There are other places where `${CMAKE_INSTALL_PREFIX}` is used. Should we replace all of them with `\$ENV{DESTDIR}${CMAKE_INSTALL_PREFIX}`? If so, probably, a new variable is required.

Also, current approach requires trailing `/` in `$DESTDIR`. Maybe that should be handled gracefully.

Reply:
Oh, I probably got it. Please discard the comment about `/`.

Is the thing that for all the other `install()` commands `DESTDIR` works out of the box, but this particular `install` requires `\$ENV{DESTDIR}` explicitly because it performs some side work and `cmake` cannot understand that it needs to append `$DESTDIR`?

Reply:
Yes. We don't need to use DESTDIR when the path is for a destination to be installed - cmake automatically resolves the correct path if DESTDIR is passed. In this case, it is important that cmake_install.cmake has $ENV{DESTDIR} so the absolute path can then be set to DIR which is then used by gen_mkldnn_compat_cmakes.cmake.

Reply:
I see, Thanks for confirming this!